docs/src/tutorial/clibraries.rst

```diff
@@ -23,11 +25,29 @@ type that can encapsulate all memory management.
 .. [CAlg] Simon Howard, C Algorithms library, http://c-algorithms.sourceforge.net/
 
 
+Download & Install [CAlg]
+==========================
+
+To download CAlg run::
+
+    $ git clone https://github.com/fragglet/c-algorithms.git
```

Comment:
I think it would be better to provide a github download link rather than requiring a `git clone`.